packages/projection-typeorm/src/operators/storeHandles.ts

```diff
@@ -230,6 +230,8 @@ const withTotalSupplies = (
     )
   );
 
+export const willStoreHandles = ({ handles }: Mappers.WithHandles) => handles.length > 0;
```

Comment:
This seems to be missing test(s)

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1200/commits/b16dab93e2ae49680cd1d6ef47b76b5823ce0cd0